packages/jest/src/enzyme-tickler.js

```diff
@@ -0,0 +1,19 @@
+const tickledCssProps = new WeakMap()
+
+export const getTickledClassName = cssProp => tickledCssProps.get(cssProp)
```

Comment:
```suggestion
export const getTickledClassName = cssProp => {
  if(!tickledCssProps.has(cssProp)) {
    throw new Error('Could not find cssProp, ensure enzyme snapshots are configured correctly - https://emotion.sh/docs/testing.');
  }
  return tickledCssProps.get(cssProp)
};
```

Reply:
Just in case we detect a shallow component, but it wasn't "tickled".

Reply:
Actually - we'd have to complicate this check because the `css` prop can "resolve" to a null (or other falsy values and:
```js
const wm = new WeakMap()
wm.set(null, {}) // throws, that's why we return for `!cssProp` in `tickle`
wm.get(null) // ok, just returns undefined
```

So I would be in favor of keeping this as-is for simplicity. I don't see when the proposed error would actually throw and it might be friendlier to the user to actually print something, rather than preventing them to work for some kind of an edge case. This is not a crucial production path after all.

Reply:
Sounds good, thanks for looking into it.